src/components/common/Text/Text.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+import './Text.css'
+
+const Text = ({ type, text }) => {
+    if (type === 'text') {
+        return <input
+            className="input input--type"
+            type={type}
+            value={text}
+        />
+    }
+
+    return <input>{text}</input>
```

Comment:
Line 14: This seems wrong time. Why am I returning an input when what we are doing is trying to create a single reusable component. 

I realize also I need to figure out a way to ask better questions so I can google research better. If you could give me tips how to improve my research skills that would be awesome!